util.go

```diff
@@ -23,3 +50,33 @@ func ParsePrice(priceStr string) (common.Fixed64, common.Fixed64, error) {
 	}
 	return entryToExitPrice, exitToEntryPrice, nil
 }
+
+func getLocationFromIP2C(ip string) (*Location, error) {
+	queryUrl := IP2CUrl + ip
+	resp, err := http.Get(queryUrl)
```

Comment:
We should always add a timeout when making request, otherwise this line may block forever. Example: https://github.com/nknorg/nkn/blob/master/api/httpjson/client/client.go#L31